stages/stage_2/compile.py

```diff
@@ -116,10 +117,10 @@ def do_com_prog(prog, macro_lookup, symbol_table, run_program):
 
     operator = prog.first()
     if operator.listp():
-        # (com ((OP) . RIGHT)) => ((c (com (q OP)) 1))
+        # (com ((OP) . RIGHT)) => (a (com (q OP)) 1)
         inner_exp = eval(prog.to([b"com",
```

Comment:
This had me confused for a second, because I read `eval` as "run this thing and return" vs. what it actually is, which is "wrap in eval opcode"

Reply:
You're right, it's badly named. This code is quite a disaster.